docs/my-website/docs/providers/snowflake.md

```diff
@@ -0,0 +1,80 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+# Snowflake
+
+Currently, Snowflake's REST API does not have an endpoint for `snowflake-arctic-embed` embedding models. If you want to use these embedding models with Litellm, you can call them through our Hugging Face provider. 
+
+Find the Arctic Embed models [here](https://huggingface.co/collections/Snowflake/arctic-embed-661fd57d50fab5fc314e4c18) on Hugging Face.
+## Supported OpenAI Parameters
+```
```

Comment:
Can you add an overview section at the top like this 
<img width="556" alt="Screenshot 2025-03-11 at 8 12 14 AM" src="https://github.com/user-attachments/assets/72097e48-a20e-4cee-9ff6-020c47b025d1" />